examples/platforms/jn5189/README.md

```diff
@@ -0,0 +1,138 @@
+# OpenThread on NXP JN5189 Example
+
+This directory contains example platform drivers for the [NXP JN5189][jn5189]
```

Comment:
missing references for both jn5189 and jn5189-dk006

Reply:
Added references.